src/main/java/com/nike/cerberus/service/S3StoreService.java

```diff
@@ -126,6 +125,20 @@ public void put(String path, String value) {
                         .collect(Collectors.toSet());
     }
 
+    /**
+     * List under a path as-if it were a folder
+     */
+    public Set<String> listUnderPartialPath(String path) {
+
+        return getKeysInPartialPath(path)
+                .stream()
+                .map(str -> StringUtils.removeStart(str, path))
```

Comment:
What do you think about adding a short a comment to each of these maps? so that it's super clear what they do